.gitignore

```diff
@@ -1,36 +1,36 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 
-# dependencies
+# Dependencies
 /node_modules
 /.pnp
 .pnp.js
 
-# testing
+# Testing
 /coverage
 
-# next.js
-/.next/
-/out/
-
-# production
+# Production
 /build
 /.search/*
 !/.search/.gitkeep
```

Comment:
These two lines can be removed and this section can be merged with `Generated files`, rest looks good.

And I think we'll need LFS (`.gitattributes`) for the images. This one might be quite complete and suitable: https://gist.github.com/webbertakken/ff250a0d5e59a8aae961c2e509c07fbc